### PR TITLE
Test spotify using an external library

### PR DESCRIPTION
**Pull request by JesusPaz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "eslint-config-airbnb": "^18.0.1",
     "eslint-config-airbnb-base": "^14.0.0",
     "mochawesome": "^4.1.0",
+    "spotify-web-api-node": "^4.0.0",
     "superagent": "^5.1.0",
```
Comment (yesid-lopez) [inline package.json]: You can't use a Spotify library, you have to call the endpoints

**Comment by yesid-lopez** [comment]
You have to call the endpoints

**Review by yesid-lopez — CHANGES_REQUESTED** [review]
You have can't use that library

**Review by renardete — APPROVED** [review]

File: test/SpotifyApi.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const request = require('request');
+const { expect, assert } = require('chai');
+const statusCode = require('http-status-codes');
+
+
+describe('Spotify Api Test', () => {
+  it('Test the Spotify access token', async () => {
```
Comment (yesid-lopez) [inline test/SpotifyApi.test.js]: If you don't use await, delete async

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+            "args": [
+                "--timeout",
+                "999999",
+                "--colors",
+                "${workspaceFolder}/test"
+            ],
+            "internalConsoleOptions": "openOnSessionStart"
+        }
+    ]
```
Comment (yesid-lopez) [inline .vscode/launch.json:17]: EOF

**Review by yesid-lopez — CHANGES_REQUESTED** [review]
Solve comments

File: .vscode/launch.json (hunk 3 above)
Comment (JesusPaz) [reply .vscode/launch.json:17]: What is the meaning of EOF? I need to delete the ".vscode" dir?

Comment (yesid-lopez) [reply .vscode/launch.json:17]: Just add an enter at the end of the file

Comment (JesusPaz) [reply .vscode/launch.json:17]: ok, thanks.

File: .travis.yml [hunk 4]
```diff
@@ -9,4 +9,6 @@ except:
...
+  - secure: al7/mMnCmdworX2CbLwD8kSw9o6wGrNQwjFmlK00bNpKTFczzIZQJN5lzeR7airEXCq4oCCZno8eRNrhwieYKDMzw8Wq2la5J/LZSCCuzNiu8D5PYjqpUeEGyODLwdtxQs9F0JkwIZDaranKst5gKVzuFKGnYNI7XYzr8tTZvmInKS+g/oZ4nNlCIjm11jc0a+ejB9OKmDuybysLZFnlQdFaiq30fNVokimK2nSCGj8WdFXsWGk8dhwfQJsNlL7YDG9WBl6V46mhEoBqsWy3hVxBzrklpujBTgznJRQLWCeJSycbZvSZsv1rnrsL9xXZ4o+qPurAko2XRsYaoNY5Y6+5lzfoyTFJepvvXkCH3diyQfroShhpyw2riWXVTbnKL6iNkqH75w0JchBJ26oGxtmkki56OV+k2gCnIjIf4RhwYgzKW87SWerYa5vuaRMj7NI/Xphrd/ByHRbvTqK7NBxypKFkUK+rNhHYXIz3F5/5jh1lQ3kRxDCPFkaogk6VeJHRccS0dKBMlrxkiaUMwDucKmjCWRAsODk5LnmVCBXWd4Tlyxq6OCfmEI3R7alaO+RuKfG3EeG/LcdMqth/IltlHGoMAQiTGc9O4GETq/Ydel+vLYIRqFmFBFXShsd5cXpNPuTdVPVxsYSl0YLDd5UchoOXW5h+B86L+7z/V34=
```
Comment (yesid-lopez) [inline .travis.yml:12]: Why do you set 3 variables if you only use 2: client id and cliend secret

Comment (JesusPaz) [reply .travis.yml:12]: I was using a GitHub access token, I'm going to delete it.

**Review by holgiosalos — APPROVED** [review]

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**